qualtran/qref_interop/bartiq_demo.ipynb

```diff
@@ -366,22 +1005,30 @@
    "id": "e864d60d-66cc-4bc7-92dc-708c2a77b614",
    "metadata": {},
    "source": [
-    "In order to get the value of these resources for a specific value of `N`, we can use Bartiq's `evaluate` method:"
+    "With the `N` parameter correctly setup, we no longer see the warning and correclty compile and print the symbolic expression for the number of T gates. An extermely useful benefit of Bartiq is being able to generate numerical values for the resources given a specific value of `N`. To do this, we can use Bartiq's `evaluate` method."
```

Comment:
```suggestion
    "With the `N` parameter correctly setup, we no longer see the warning and correctly compile and print the symbolic expression for the number of T gates. Now we can generate numerical values for the resources given a specific value of `N`. To do this, we can use Bartiq's `evaluate` method."
```
I wouldn't say it's an "extremely useful benefit", as it's something you can do with symbolic expressions in general.